homeassistant/components/home_connect/binary_sensor.py

```diff
@@ -119,19 +120,61 @@ async def async_setup_entry(
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up the Home Connect binary sensor."""
+    added_entities: set[str | None] = set()
```

Comment:
why or None?

Reply:
Good question, I think it is a mistake or it was something but now is stale code.